hedera-node/hedera-app/src/main/java/com/hedera/node/app/bbm/utils/ThingsToStrings.java

```diff
@@ -263,6 +303,105 @@ public static boolean toStringOfFcTokenAllowanceIdSet(
         return true;
     }
 
+    public static boolean toStringOfApprovalForAllAllowances(
+            @NonNull final StringBuilder sb, @Nullable List<AccountApprovalForAllAllowance> approvals) {
+        if (approvals == null || approvals.isEmpty()) {
+            return false;
+        }
+
+        final var orderedApprovals = approvals.stream().sorted().toList();
+        sb.append("(");
+        for (final var approval : orderedApprovals) {
+            toStringOfApprovalForAllAllowance(sb, approval);
+            sb.append(",");
+        }
+        sb.setLength(sb.length() - 1);
+        sb.append(")");
```

Comment:
Haha, it happens 😄 

I started to update the method but it looks like there is a difference between the approaches. In `toStringOfIntArray` we are iterating and directly appending the ints but here we are calling `toStringOfApprovalForAllAllowance`(which calls `toStringOfTokenId` and `toStringOfAccountId`) and all these methods are appending to the StringBuilder. So it seems to me that it would be not very straightforward to refactor all the methods(and the methods that are calling them)

Reply:
ok